content/docs/SUSHI/running/_index.md

```diff
@@ -15,27 +15,102 @@ This documentation assumes you have a SUSHI-compliant project structure and conf
 SUSHI is executed from the command line. The general form of the SUSHI execution command is as follows:
 
 ```shell
-{{< terminal >}} sushi {specification-directory} {options}
+{{< terminal >}} sushi {options}
 ```
 
+Use `sushi --version` and `sushi --help` to get basic information about the current SUSHI version.
+
+{{% small-pageinfo color="primary" %}}
+<span class="tag">SUSHI 3.0</span>For SUSHI 3.0.0 and later, the general form of the SUSHI execution command is:
+
+```shell
+$ sushi {command} {options}
+```
+
+Supported commands are `build`, `init`, `update-dependencies`, and `help` (which replaces `sushi --help`).
+{{% /small-pageinfo %}}
+
+### SUSHI Commands
+
+SUSHI provides various commands to use with FSH projects. The following sections provide more details on each.
+
+#### `build`
+
+The `build` command is used to build a SUSHI project. In SUSHI 2.x, it is the default command and can be used as follows:
+
+```shell
+{{< terminal >}} sushi {fsh-project-directory} {options}
+```
+
+{{% small-pageinfo color="primary" %}}
+<span class="tag">SUSHI 3.0</span>For SUSHI 3.0.0 and later, use the `build` command:
+
+```shell
+$ sushi build {fsh-project-directory} {options}
+```
+
+{{% /small-pageinfo %}}
+
 where options include the following (in any order):
 
 ```text
--d, --debug         output extra debugging information
--h, --help          output usage information
--i, --init          initialize a SUSHI project
--o, --out <out>     the path to the **fsh-generated** folder
--p, --preprocessed  output FSH produced by preprocessing steps
---require-latest    indicate that SUSHI should halt if the latest SUSHI version is not installed
--s, --snapshot      generate snapshot in Structure Definition output (default: false)
--v, --version       print SUSHI version
+-o, --out <out>          the path to the output folder
+-d, --debug           output extra debugging information
```

Comment:
It would be nice to align the output. Technically you can probably remove three spaces from each line because the `-l --log-level` line was longer, or you can just add three spaces to `-d --debug`

```suggestion
-d, --debug              output extra debugging information
```